src/phoenix/server/api/dataloaders/span_descendants.py

```diff
@@ -52,8 +52,7 @@ async def _load_fn(self, keys: List[Key]) -> List[Result]:
         stmt = (
             select(descendant_ids.c[root_id_label], models.Span)
             .join(descendant_ids, models.Span.id == descendant_ids.c.id)
-            .join(models.Trace)
-            .options(contains_eager(models.Span.trace))
+            .options(joinedload(models.Span.trace, innerjoin=True).load_only(models.Trace.trace_id))
```

Comment:
We can use `joinedload` here since we don't use the joined trace elsewhere except in the selected span. This is preferable IMO because `contains_eager` has unintuitive behavior if the explicit join on `traces` is removed.